identity-wallet/bindings/ConnectionSorting.ts

```diff
@@ -0,0 +1,3 @@
+// This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
+
+export type ConnectionSorting = { "name_az": { reverse: boolean | null, } } | { "first_interacted_new_old": { reverse: boolean | null, } } | { "last_interacted_new_old": { reverse: boolean | null, } };
```

Comment:
Should be generated as `reverse?: boolean` instead of `reverse: boolean | null` (same as in `UpdateSortingPreference.ts`). It's not necessary to fix it across the _entire_ repo in this PR, but only for the newly introduced bindings.